src/compiler/crystal/compiler.cr

```diff
@@ -18,6 +18,7 @@ module Crystal
   # optionally generates an executable.
   class Compiler
     CC = ENV["CC"]? || "cc"
+    CL = "cl"
```

Comment:
@bcardiff Because, just like on Unix, calling the linker directly would require you to manually pass the correct C/base libraries to link. On Windows, it's more complicated than Unix; in addition to `kernel32.lib`, you need to link in the proper C library using its full path, depending on the architecture and debug vs release mode.